packages/@o3r/rules-engine/src/engine/ruleset-executor.ts

```diff
@@ -317,6 +317,7 @@ export class RulesetExecutor {
       id: this.ruleset.id,
       validityRange: this.ruleset.validityRange,
       linkedComponent: this.ruleset.linkedComponent,
+      linkedComponents: this.ruleset.linkedComponents,
```

Comment:
Could plan the future migration by using array only
```suggestion
      linkedComponents: this.ruleset.linkedComponents || this.ruleset.linkedComponent && [this.ruleset.linkedComponent],
```

Reply:
But I support both usecases. I can have a linkedComponent and linkedComponents too.
Eventually I could do an array:
`Array.from(new Set([this.ruleset.linkedComponent, ...this.ruleset.linkedComponents])).filter(c => !!c)`
?